src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -509,6 +515,55 @@ public Task<Block> GetAsync(uint256 hash)
             return task;
         }
 
+        /// <summary>
+        /// Get multiple blocks from the database by their block hashes.
+        /// </summary>
+        /// <param name="hashes">The block hashes.</param>
+        public Task<List<Block>> GetBlocksAsync(List<uint256> hashes)
+        {
+            this.logger.LogTrace("({0}:'{1}')", nameof(hashes.Count), hashes.Count);
```

Comment:
just `{1}` instead of `'{1}'` for integer values (such as count)

Reply:
Done.